### ISSUE TITLE
connection problem

### ISSUE THREAD

**Issue by Universalera** [HUSTSYJ/DA_dahazing#8]
raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=8097): Max retries exceeded with url: /events('<urllib3.connection.HTTPConnection object at 0x7f45a624db70>: Failed to establish a new connection: [Errno 111] Connection refused',))

**Comment by anothersin** [comment]
did you slove this problem?

**Comment by Universalera** [comment]
> did you slove this problem?

yeah i had already solved it. just download required package and then install it.

**Comment by anothersin** [comment]
> > did you slove this problem?
> 
> yeah i had already solved it. just download required package and then install it.

i have installed it. however it still exits
![image](https://user-images.githubusercontent.com/23079250/94650575-913a6d80-0329-11eb-8a49-78eb0483fda9.png)

**Comment by Universalera** [comment]
> > > did you slove this problem?
> > 
> > 
> > yeah i had already solved it. just download required package and then install it.
> 
> i have installed it. however it still exits
> ![image](https://user-images.githubusercontent.com/23079250/94650575-913a6d80-0329-11eb-8a49-78eb0483fda9.png)

oh i forgot to say that u need to run it

**Comment by hello-trouble** [comment]
Thank you for your issue. But I am insterested in which packages
 need to be installed?
